### PR TITLE
Main guimanager

### PR DESCRIPTION
**Pull request by SurfaceS**
Swing separation part 2.

### CODE REVIEW

File: src/main/java/net/pms/PMS.java [hunk 1]
```diff
@@ -547,19 +531,20 @@ private boolean init() throws Exception {
 		 * would be cleaner and could support other lifecycle
 		 * notifications (see above).
 		 */
-		FrameAppender.setFrame(frame);
+		//FrameAppender.setFrame(frame);
```
Comment (SubJunk) [inline src/main/java/net/pms/PMS.java:534]: Why is this commented?

Comment (SurfaceS) [reply src/main/java/net/pms/PMS.java]: fixed

**Comment by SurfaceS** [comment]
removed `updateRenderer` from `PMS`, `IGui`/`LookFrame` and `StatusTab` classes.

In fact, `PMS` call the `IGui`/`LookFrame` `updateRenderer(renderer)` function.
Then `IGui`/`LookFrame` call the `StatusTab` `updateRenderer(renderer)` function.
Then `StatusTab` call the `renderer.updateRendererGui()` function.
As a `IRendererGuiListener`, `RendererItem.updateRenderer(renderer)` is called from the `renderer.updateRendererGui()`.

Directly use the `renderer.updateRendererGui()` instead as we have it from start.

**Review by SubJunk — APPROVED** [review]

**Comment by SubJunk** [comment]
@SurfaceS After merging I noticed the Logs tab is broken

**Comment by SurfaceS** [comment]
> @SurfaceS After merging I noticed the Logs tab is broken

I'll look at that now.

**Comment by SurfaceS** [comment]
Oh, I maybe understand why.
The file [src/main/external-resources/logback.xml](https://github.com/UniversalMediaServer/UniversalMediaServer/pull/3293/files#diff-aa3c4e4874b50aa5264e6f2cdaaa94decb102f184d4776f7823b654009a4055c) was changed.
So if you run it without install, it keep the old logback.xml.
I had a blank log when until I install (or replace the logback file).

**Comment by SubJunk** [comment]
@SurfaceS you're right, that was it. Thanks for looking into it

**Pull request merged**